### PR TITLE
Simple kintoization

### PR DESCRIPTION
**Pull request by n1k0**
This stores shows data on [Kinto](https://kinto.readthedocs.org/) in the simplest possible way. Basically shows are stored in a dedicated `myshows` collection on users' personal bucket, provided they're authenticated using their [Firefox Account](https://accounts.firefox.com/).

Later we obviously want to store show entries in records, but let's iterate here.

### CODE REVIEW

File: Model.elm [hunk 1]
```diff
@@ -117,18 +127,51 @@ onStoreLoaded json =
...
+          , shows = []
+          , currentOrderBy = TitleAsc
+          , currentGenre = Nothing
+          , allGenres = extractAllGenres []
+          , formData = initFormData
+          , formErrors = []
+          , formEdit = Nothing
+          }
+        , fetchBackup authToken
+        )
```
Comment (magopian) [inline Model.elm:186]: I see you're using the `Model ! []` syntactic sugar below, don't you want to also use it here?

File: Model.elm [hunk 2]
```diff
@@ -290,18 +359,18 @@ update msg ({ shows, formData } as model) =
                     | shows = updatedShows
                     , allGenres = extractAllGenres updatedShows
                   }
-                , saveShows updatedShows
+                , saveBackup authToken updatedShows
```
Comment (magopian) [inline Model.elm:374]: Do you want to use `Model ! []` everywhere?

File: Model.elm [hunk 3]
```diff
@@ -101,11 +116,6 @@ dummyShows =
     ]
 
 
-subscriptions : Model -> Sub Msg
-subscriptions =
-    always <| Store.load onStoreLoaded
-
-
 onStoreLoaded : Decode.Value -> Msg
```
Comment (magopian) [inline Model.elm]: Do you want to get rid of that unused piece of code now?

**Review by magopian — COMMENTED** [review]
This looks excellent ;) Good usage of the elm-kinto library, happy that it's useful!

File: Model.elm (hunk 2 above)
Comment (n1k0) [reply Model.elm:374]: Not really because it gets super ugly with large statements once indented by elm-format.

**Pull request merged**